docs/guests.md

```diff
@@ -122,9 +122,55 @@ On OpenBSD, the xen drivers are also already part of the kernel. The `install.sh
 For OpenBSD search [the forum](https://xcp-ng.org/forum). See for example [this thread](https://xcp-ng.org/forum/topic/2582/guest-tools-for-openbsd).
 :::
 
-## FreeNAS/TrueNAS
+## TrueNAS 12.0-Release
+TrueNAS 12.0-Release does not have `xe-guest-utilities` installed. But the forthcoming TrueNAS 12.0-U1 and later should have them installed by default and should not require any user action. 
 
-FreeNAS is a locked-down version of FreeBSD, with many packages disabled to ensure a more stable environment for the fileserver. `xe-guest-utilities` is part of the packages that are **not** available in FreeNAS. But because it's based on FreeBSD, the packages from that OS can be installed, at your own risk. This is not a big issue for this particular package, because it's a _leaf_ in the chain of dependencies - nothing in FreeNAS depends on it.
+Meanwhile, if you are on TrueNAS 12.0-Release here are the installation instructions for `xe-guest-utilities`:
+
+To install it, you just have to enable the FreeBSD repo first:
+
+```bash
+# nano /usr/local/etc/pkg/repos/local.conf
```

Comment:
The sed commands that are available in the current documentation are more fast and less error-prone to be run.